<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QCheckBox" name="checkBox_Wednesday">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>240</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Wednesday</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Thursday">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Thursday</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Friday">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Friday</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Saturday">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Saturday</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Sunday">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>400</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Sunday</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Tuesday">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Tuesday</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Monday">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>120</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Monday</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="hours">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>64</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="minutes">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>64</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTimeEdit" name="setTime">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>118</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="setDateandTime">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Day And Time</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hideSetTime">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="setMusic">
   <property name="geometry">
    <rect>
     <x>88</x>
     <y>250</y>
     <width>161</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>SetMusic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hideMusic">
   <property name="geometry">
    <rect>
     <x>38</x>
     <y>410</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>hideMusic</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>290</y>
     <width>341</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
